.github/workflows/UnitTests.yml

```diff
@@ -29,6 +29,8 @@ jobs:
               run: |
                   export CHANGED_GLOBAL_FILES=$(echo "${{ steps.files.outputs.all }}" | grep -v "^packages")
                   echo "::set-output name=arg::$(if [ "${CHANGED_GLOBAL_FILES}" = "" ] && ${{ github.event_name == 'pull_request' }}; then echo '--since origin/${{ github.base_ref }}'; else echo ''; fi)"
+            - name: "Validating widgets' copyright"
```

Comment:
The consequence of this is that in the case of failing builds on master at the moment that the new year starts, we will have to fix master in the automated PR. Not sure if that's desirable...

Reply:
I'm not sure if I understand. `master`(failing) -> new year -> run cron+bump -> PR+commit(code changeset)

Fix broken master, get that merged then rebase the automated PR, all good?

Reply:
version validation is addressed [here](https://github.com/mendix/widgets-resources/blob/6e6c7293a5f3bad60d089e26d8417e65a7a85cdf/.husky/pre-commit#L4), i will close this convo